torch/__init__.py

```diff
@@ -270,22 +270,30 @@ def set_default_tensor_type(t):
 
 
 def set_default_dtype(d):
-    r"""Sets the default floating point dtype to :attr:`d`. This type will be
-    used as default floating point type for type inference in
-    :func:`torch.tensor`.
+    r"""Sets the default floating point dtype to :attr:`d`.
+    This type is also used:
```

Comment:
this says "also used" but it's sort of defined by how it's used, i.e. "sets the default floating point dtype" doesn't really mean anything outside of how it' s used.  I'd just change this to something like:
"This dtype is used as:"

Reply:
updated